docs/visual-studio-extensibility/nuget-api-in-visual-studio.md

```diff
@@ -335,207 +419,203 @@ public interface IRegistryKey
     }
 ```
 
-## IVsNuGetFramework interface
```

Comment:
Why are we removing this? I don't see it added again.

Reply:
My thought was that this isn't the dotnet/api reference. We're kind of treating it as a reference document (more so than any of our other NuGet docs), but at the same time we wrote at the top of the document that callers can MEF import these interfaces. `IVsNuGetFramework` can't be MEF imported. Technically `IVsPathContext` can't either, but I left it because of the following reason:

My intention was to document on this page the APIs that customers (VS extension developers) will care about. A way to use browser search to find interesting APIs like how to get the list of installed packages, how to install or uninstall, how to get configured package source, etc. I don't think `IVsNuGetFramework`'s properties or XMLDoc comments are interesting/insightful, or will help customers using find to search for an API they're looking for.

On the other hand, if they decide that one of the other interfaces has a method that they'd like to use, their code editor's intellisense or "go to definition" disassembly viewer, will show them what properties `IVsNuGetFramework` has, which is sufficient to empower them to achieve their goal.

Anyway, that was my intent, and why I deleted a bunch of interfaces that are not MEF importable, but are POCO definitions used as input/return values. Let me know if you don't agree with my intent, I can put it back.

Reply:
I don't have too strong of an opinion in either direciton.